src/04-modality-specific-files/06-physiological-and-other-continuous-recordings.md

```diff
@@ -133,12 +133,12 @@ sub-01_task-cuedSGT_run-1_echo-3_bold.nii.gz
-| Manufacturer           | RECOMMENDED           | [string][]    | Manufacturer of the system used to record the physiological data                                                                                                                                                 |
-| ManufacturersModelName | RECOMMENDED           | [string][]    | Manufacturer's designation of the system used to record the physiological data                                                                                                                                   |
-| SoftwareVersions       | RECOMMENDED           | [string][]    | Manufacturer's designation of the acquisition software.                                                                                                                                                          |
-| DeviceSerialNumber     | RECOMMENDED           | [string][]    | The serial number of the equipment that produced the composite instances. A pseudonym can also be used to prevent the equipment from being identifiable, as long as each pseudonym is unique within the dataset. |
+| **Key name**           | **Requirement level** | **Data type** | **Description**                                                                                                                                                                                                 |
+| ---------------------- | --------------------- | ------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Manufacturer           | RECOMMENDED           | [string][]    | Manufacturer of the system used to record the physiological data                                                                                                                                                |
+| ManufacturersModelName | RECOMMENDED           | [string][]    | Manufacturer's designation of the system used to record the physiological data                                                                                                                                  |
+| SoftwareVersions       | RECOMMENDED           | [string][]    | Manufacturer's designation of the acquisition software.                                                                                                                                                         |
+| DeviceSerialNumber     | RECOMMENDED           | [string][]    | The serial number of the equipment that produced the physiological data. A pseudonym can also be used to prevent the equipment from being identifiable, as long as each pseudonym is unique within the dataset. |
```

Comment:
I think I would keep the text for the different fields (here `DeviceSerialNumber`) the same everywhere. "measurements" would work here as well.

Reply:
Here, I just tried to make the minimal change, and the lines above already had `physiological data`. Do you want me to make it measurements in all modality files? Including fMRI?

Reply:
That's something to discuss together with @tsalo: If we want to render all these metadata fields from the schema, the schema entry should/must be consistent.

Reply:
Yes, thank you. The "canonical" metadata definitions in the schema (see #774) are going to just use the generic definition (i.e., "measurement" instead of "physiological data"), so it would be preferable to just stick with "measurement" in the current tables.

Reply:
Thanks @tsalo